code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -741,8 +741,7 @@
 	return ..() | update_flags
 
 /datum/reagent/fliptonium/reaction_mob(mob/living/M, method=REAGENT_TOUCH, volume)
-	if(method == REAGENT_INGEST || method == REAGENT_TOUCH)
-		M.SpinAnimation(speed = 12, loops = -1, parallel = FALSE)
+	M.SpinAnimation(speed = 12, loops = -1, parallel = FALSE)
```

Comment:
Why is the check removed altogether for fliptonium?

Reply:
because the only existing methods are `REAGENT_INGEST` and `REAGENT_TOUCH`, meaning there can be no other option for that variable